CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [6.1.2] - TBD
+### Added
+- Added apiary_logs_retention_days variable that sets the default retention period of the apiary cloudwatch group. The default is 30 days.
```

Comment:
should we name the variable apiary_cloudwatch_logs_retention_days? to avoid confusion with s3 logs retention

Reply:
ok, makes sense. updating.

Reply:
Does the same apply here? https://github.com/ExpediaGroup/apiary-federation/pull/75#pullrequestreview-409293184

Reply:
I can't say. For me adding cloudwatch in the name only makes sense for apiary-data-lake but you may want to make variable names consistent accross the different components.